File: BikeShop.Frontend/webclient/src/entities/entities/Schedule/ScheduleHistory.ts
export interface ScheduleHistory {
    id: number
    createdAt: string
    updatedAt: string
    enabled: boolean

    action: string;
    actionUser: string;
    actionUserFIO: string;
    actionTargetUser: string;
    actionTargetUserFIO: string;
    isHolydayPrev: boolean;
    timeStartPrev: string;
    timeFinishPrev: string;
    isHolydayActual: boolean;
    timeStartActual: string;
    timeFinishActual: string;
    itemId: number;
}

File: BikeShop.Backend/BikeShop.Shop/BikeShop.Shop.Domain/Entities/ScheduleHistory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BikeShop.Shop.Domain.Entities
{
     public class ScheduleHistory : BaseEntity
    {
        public string Action { get; set; }
        public Guid ActionUser { get; set; }
        public string ActionUserFIO { get; set; }
        public Guid ActionTargetUser { get; set; }
        public string ActionTargetUserFIO { get; set; }

        public bool IsHolydayPrev { get; set; }
        public DateTime TimeStartPrev { get; set; } = new DateTime();
        public DateTime TimeFinishPrev { get; set; } = new DateTime();

        public bool IsHolydayActual { get; set; }
        public DateTime TimeStartActual { get; set; }=  new DateTime();
        public DateTime TimeFinishActual { get; set; } = new DateTime();

        public int ItemId { get; set; }
        public int ShopId { get; set; }
    }
}
